pmlc/dialect/tile/builder.cc

```diff
@@ -343,6 +343,13 @@ std::vector<Value> TileBuilder::GetTupleElements(Value value) {
   throw std::runtime_error("Expected TupleOp");
 }
 
+Value TileBuilder::MakeScalarConstantOp(uint64_t value) {
+  IVLOG(5, "TileBuilder::MakeScalarConstantOp> " << value);
+  auto type = impl->builder.getType<ScalarType>(DataType::i32);
```

Comment:
Maybe use `DataType::u64` instead? For now, this is consistent with the `int64_t` case, which also uses `DataType::i32`